cmd/operator/api/v1alpha1/kubearchiveconfig_webhook.go

```diff
@@ -4,20 +4,20 @@
 package v1alpha1
 
 import (
-	"context"
 	"errors"
+	"fmt"
 
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 
 	cel "github.com/kubearchive/kubearchive/pkg/cel"
 )
 
+const kubearchiveResourceName = "kubearchive"
```

Comment:
should we maybe have the name provided through an environment variable?

Reply:
Why? What advantage does that provide?

Reply:
just in case that the KubeArchive deployer wanted to override the resourcename with a different one, but it's not that important (I already approved the PR, I just thought that it will provide more versatility with little work), but I think that probably no one will care about it so feel free to ignore the suggestion

Reply:
> just in case that the KubeArchive deployer wanted to override the resourcename with a different one,

But doesn't that go completely against the whole point of this PR?  I guess you're changing or interpreting it to mean "force all `KubeArchiveConfig` resources to have the same name"? I was strictly adhering to "force all `KubeArchiveConfig` resources to be named 'kubearchive'".

If this becomes an problem we can open a new issue to support what I think you are looking for.